### ISSUE TITLE
Java调优经验谈

### ISSUE THREAD

**Issue by ermingzhu** [superhj1987/superhj1987.github.io#22]
怎么找出瓶颈呢？根据哪些面板的哪些参数进行分析

**Comment by superhj1987** [comment]
@ermingzhu 瓶颈一般来说需要压测，或者说在出现问题时在大部分指标正常的情况下，某一个/几个指标异常则异常指标对应的即为瓶颈，如：用户量不断上升的过程中，如果CPU使用率还维持在50%左右，但是IO负载（如ioutil）已经90%以上了，那么此时IO性能是瓶颈。具体问题需要具体分析。

**Comment by bneliao** [comment]
使用火焰图来找到系统cpu的使用瓶颈，比较常见

**Comment by superhj1987** [comment]
@bneliao 是的。如果定位到是CPU是瓶颈，进一步的可以使用火焰图来定位是哪一个方法调用是瓶颈。JProfiler、JVM-Profiler都可以使用。
